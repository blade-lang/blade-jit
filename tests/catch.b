catch {
  var a = 20
  raise new Error('Something happened')
} /*as e {
  echo e
}*/ then {
  echo 'Cleaning up'
}