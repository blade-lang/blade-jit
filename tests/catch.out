^^^
Error: Something happened
	at <nim> @.script(tests/catch.b:3:23-60)